SONIC-DEV/sonic-buildimage
# [Fishbones/phalanx] Update switchboard driver to fix EEPROM not detected

**- What I did**
Update the Celestica platform submodule pointer for fixing EEPROM not detected issue when running sfputil multi-thread stress test.

**- How I did it**
Applied the following changes to Fishbone32, Fishbone48, and Phalanx:
1. Fix race condition in Switchboard CPLD port XCVR resource.

Applied the following changes to Phalanx:
1. Fix one PCA9548 cannot be reset on each line card.

**- How to verify it**
1. Plugs some QSFPs modules to UUT.
2. Run the stress test script to continuously access port EEPROM using sfputil and check there was no "EEPROM not detected" on the plugged modules.

See JIRA: [CAFP-115](http://jira.cn-csh.celestica.com:8080/browse/CAFP-115)
